tests/e2e/api/controllers/monitoring.spec.js

```diff
@@ -1,5 +1,6 @@
-const { expect } = require('chai');
-
+const chai = require('chai');
```

Comment:
Can you explain the logic behind the change away from the `expect` function to working through the `chai` object like this?